inst/include/dust/distr/binomial.hpp

```diff
@@ -23,22 +23,26 @@ double binomial_inversion(double n, double prob, RNG& generator) {
   return num_geom;
 }
 
-inline double stirling_approx_tail(double k) {
-  static double kTailValues[] = {0.0810614667953272,  0.0413406959554092,
+template <typename real_t>
+real_t stirling_approx_tail(real_t k) {
+  static real_t kTailValues[] = {0.0810614667953272,  0.0413406959554092,
```

Comment:
This function may want to stay as a double, but I'm not sure. In the tf code it isn't templated

Reply:
yup, reverted now - I should have checked the tf code first!